src/IceRpc/Transports/Tcp/Internal/TcpListener.cs

```diff
@@ -32,13 +35,20 @@ internal sealed class TcpListener : IListener<IDuplexConnection>
         }
         catch (SocketException exception)
         {
+            if (exception.SocketErrorCode == SocketError.OperationAborted)
```

Comment:
I updated Tcp/Coloc listeners to do the same. Alternatively we can move this to Slic and don't update Tcp/Coloc behavior.